protocol_rfcs/README.md

```diff
@@ -16,10 +16,11 @@ Here is the history of all the RFCs propose/accepted/rejected since Feb 6, 2024,
 
 ### Proposed RFCs
 
-| Date proposed | RFC file | Github issue | RFC title |
-|:-|:-|:-|:-|
-| 2023-02-02 | [in-commit-timestamps.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/in-commit-timestamps.md) | https://github.com/delta-io/delta/issues/2532 | In-Commit Timestamps |
-| 2023-02-09 | [type-widening.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/widening.md) | https://github.com/delta-io/delta/issues/2623 | Type Widening |
+| Date proposed | RFC file                                                                                                         | Github issue                                  | RFC title                     |
+|:-|:-----------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:------------------------------|
+| 2023-02-02 | [in-commit-timestamps.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/in-commit-timestamps.md)   | https://github.com/delta-io/delta/issues/2532 | In-Commit Timestamps          |
+| 2023-02-09 | [type-widening.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/widening.md)                      | https://github.com/delta-io/delta/issues/2623 | Type Widening                 |
+| 2023-02-09 | [vacuum-protocol-check.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/vacuum-protocol-check.md) | https://github.com/delta-io/delta/issues/2630 | Enforce Vacuum Protocol Check |
```

Comment:
managed-commit RFC seems to be missing from this list. Could you add the same?

Reply:
Done